Orange/widgets/regression/owsgdregression.py

```diff
@@ -130,6 +129,9 @@ def __init__(self):
         self._on_func_changed()
 
         self.apply()
+        
+
+        
```

Comment:
Prefer not to introduce extra vertical whitespace when it's not needed. This is also not PEP8-compliant, try by running:
```bash
$ sudo apt-get install pep8
$ pep8 Orange/widgets/regression/owsgdregression.py
```